packages/labs/context/src/lib/helper.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ContextRequestEvent} from './context-request-event.js';
+import {Context} from './create-context.js';
+
+/**
+ * A helper method for use in situations where a web component is isolated
+ * from the parent that provides the context.
+ *
+ * This helper provides static values for the specified contexts. It does
+ * not support updates via subscribe.
```

Comment:
This doc is missing some important context I think. What actually is this function? How do you use it?